Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Storage/CachedQueries/Merchant/MerchantServiceConfig.hs

```diff
@@ -40,6 +41,10 @@ findByMerchantIdAndService id serviceName =
     Just a -> return . Just $ coerce @(MerchantServiceConfigD 'Unsafe) @MerchantServiceConfig a
     Nothing -> flip whenJust cacheMerchantServiceConfig /=<< Queries.findByMerchantIdAndService id serviceName
 
+-- FIXME this is temprorary solution for backward compatibility
```

Comment:
Can you please add cached query for this too, as this query will still be active for few days as full rollout of new api may take time